cpp/zotero_harvester.cc

```diff
@@ -152,7 +155,14 @@ void ParseCommandLineArgs(int * const argc, char *** const argv, CommandLineArgs
             return; // intentional early return
         }
         case CommandLineArgs::SelectionMode::JOURNAL:
-            commandline_args->selected_journals_.emplace(current_arg);
+            if (StringUtil::StartsWith(current_arg, "--config-overrides=")) {
```

Comment:
This should be moved to the top, in the While loop with the rest of the optional params.